packages/@aws-cdk/aws-stepfunctions-tasks/test/stepfunctions/integ.invoke-activity.ts

```diff
@@ -42,6 +42,13 @@ class InvokeActivityStack extends cdk.Stack {
     const finalStatus = new tasks.StepFunctionsInvokeActivity(this, 'Get Final Job Status', {
       activity: checkJobActivity,
       inputPath: '$.guid',
+      parameters: {
+        'input.$': '$',
+        'stringArgument': 'inital-task',
+        'numberArgument': 123,
+        'booleanArgument': true,
+        'arrayArgument': ['a', 'b', 'c'],
```

Comment:
Something along the lines of
```suggestion
        'arrayArgument': ['a', 'b', 'c'],
        'jsonPath': sfn.JsonPath.stringAt('$.status'),

Reply:
Added!